site/layouts/blog/list.html

```diff
@@ -9,6 +9,7 @@ <h1 class="text--white">{{ .Title }}</h1>
       <div class="flex on-sm-stack">
         <div class="col-2-3 pr-30">
             {{ range .Pages }}
+             {{ if not .Params.unlisted }}
```

Comment:
Better to be more specific that only applies to the blog index (e.g. it will still be listed in tags).